### PR TITLE
Add semi-test to check md5sum of fingerprint file on resin-root partion

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -481,4 +481,29 @@ if (options.interactiveTests) {
...
     })
   }
+
+  ava.test.skip('ResinOS image: Check host OS fingerprint file on resin-root partition', async (test) => {
+    return utils.runManualTestCase(test, {
+      prepare: [ 'Download a new image from the dashboard and do not provision it' ],
+      do: [
+        'Set up a loop device by running: "losetup --show --find --partscan resin.img"',
+        'Create a temporary directory by running: "mkdir /tmp/resin-root"',
+        'Mount the filesystem by running: "mount /dev/loop*** /tmp/resin-root", replacing with the appropriate loop device',
```
Comment (horia-delicoti) [inline lib/main.js]: Choose what partition to mount

File: lib/main.js [hunk 2]
```diff
@@ -481,4 +481,29 @@ if (options.interactiveTests) {
       })
     })
   }
+
+  ava.test.skip('ResinOS image: Check host OS fingerprint file on resin-root partition', async (test) => {
```
Comment (jviotti) [inline lib/main.js]: Any reason to skip this test?

Comment (horia-delicoti) [reply lib/main.js]: @jviotti You have to run md5sum for every file from the fingerprint file. This is not feasible for manual testing.

Comment (jviotti) [reply lib/main.js]: In that case, this test is not runnable, so can we create a GH issue instead, and handle it directly with code?

**Comment by telphan** [comment]
This will be superseded

**Pull request closed without merging**